http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/utils/NettyUtils.java

```diff
@@ -388,4 +395,45 @@ public static void runAndLogError(NettyClientLogger log, String errorMsg, Functi
             log.error(null, () -> errorMsg, e);
         }
     }
+
+    //  ALPN supported backported in u251
+    //  https://bugs.openjdk.org/browse/JDK-8242894
+    public static boolean isAlpnSupported(SslProvider sslProvider) {
+        if (sslProvider != SslProvider.JDK) {
+            return true;
+        }
+
+        try {
+            SSLContext context = SSLContext.getInstance("TLS");
```

Comment:
Just to double check, did we test this behavior (manually) with older java version?

Reply:
Tested on Jdoodle with JDK `1.8.0_66`, fails as intended and succeeds with Java 9 and above. Updated to catch  `PrivilegedActionException` instead of `NoSuchMethodException`